R/refineCenters.R

```diff
@@ -0,0 +1,143 @@
+#' @title Refine Peak Centers
+#'
+#' @description
+#' This function refines the center values of a peak by weighting the y values
+#' in the neightbourhood that belong most likely to the same peak.
+#'
+#' @param x `numeric`, i.e. m/z values.
+#' @param y `numeric`, i.e. intensity values.
+#' @param p `numeric`, peak centers.
```

Comment:
So, `p` is then an m/z value? I assume it is one of the values in `x`? Would it make sense that `p` would be an `integer` defining where in `x` the peaks are?

Reply:
`p` is the peak index. Will change the documentation.